lib/Delighted/AutopilotConfiguration.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Delighted;
+
+class AutopilotConfiguration extends Resource
+{
+    protected static $path = 'autopilot';
+
+    public static function retrieve($platform, Client $client = null)
+    {
+        if (is_null($client)) {
+            $client = Client::getInstance();
+        }
+
+        $response = $client->get(self::$path . '/' . urlencode($platform));
```

Comment:
Do we need a `validatePlatform` sort of thing in here, like you've put in `AutopilotMembership`?

Reply:
We could, but was thinking we could rely on the backend to validate this, which has the slight advantage of needing less configuration in the client code. I also removed the `AutopilotMembership.validatePlatform()` function to favor dedicated classes to mirror the other clients.

```javascript
// GET 'https://api.delighted.com/v1/autopilot/emailz'
{
    "status": 404,
    "message": "Autopilot for emailz does not exist."
}
```

Reply:
Cool, I was just noting the difference in validation between the two, but it sounds like they are now the same. 😄